tools/qontract_cli.py

```diff
@@ -478,6 +478,25 @@ def bot_login(ctx, cluster_name):
     print(f"oc login --server {server} --token {token}")
 
 
+@get.command()
+@click.argument('account_name')
+@click.pass_context
+def aws_creds(ctx, account_name):
+    settings = queries.get_app_interface_settings()
+    secret_reader = SecretReader(settings=settings)
+    accounts = queries.get_aws_accounts()
+    accounts = [a for a in accounts if a['name'] == account_name]
+    if len(accounts) == 0:
+        print(f"{account_name} not found.")
+        sys.exit(1)
+
+    account = accounts[0]
+    secret = secret_reader.read_all(account['automationToken'])
+    print(f"export AWS_REGION={account['resourcesDefaultRegion']}")
+    print(f"export AWS_ACCESS_KEY_ID={secret['aws_access_key_id']}")
+    print(f"export AWS_SECRET_ACCESS_KEY={secret['aws_secret_access_key']}")
```

Comment:
This is very neat, can we put in the help of this command that `$(qontract-cli aws_creds --account-name foo")` will set up the environment as expected?

Reply:
done